server/src/routes/api/v1/websitesCommentsRouter.js

```diff
@@ -10,10 +11,12 @@ websitesCommentsRouter.post("/", async (req, res) => {
   try {
     const userId = req.user.id;
     const newComment = await Comment.query().insert({ rating, comment, userId, websiteId });
-
     return res.status(201).json({ comment: newComment });
   } catch (error) {
     console.error(`ERROR ${error.message}`);
```

Comment:
remove console logs from final code - this is fine to add while developing on the feature